frontend/src/pages/profile.js

```diff
@@ -1,10 +0,0 @@
-import React from "react";
```

Comment:
Aha so you renamed the file from .js to .jsx yes?

Reply:
yessss